README.md

```diff
@@ -1,5 +1,15 @@
 # boats.py
-The official discord.boats API wrapper for Python
+The official [discord.boats](https://discord.boats) API wrapper for Python
+
+# Installation
+## Install via pip (Recommended)
+```pip install boats.py```
+## Install from source
+```
+git clone https://github.com/DuckMasterAl/boats.py
+cd boats.py
+pip install -R requirements.txt
```

Comment:
Do we really need a capital R here?

Reply:
It can be lowercase. It just needs to be an R for the requirements file.
```pip install [options] -r <requirements file>```